tests/tt_eager/python_api_testing/sweep_tests/pytests/tt_dnn/wormhole/test_bert_ternary.py

```diff
@@ -22,9 +22,13 @@ def run_ternary_bert_tests(
 ):
     torch.manual_seed(data_seed)
 
-    x = torch.Tensor(size=input_shapes[0]).uniform_(-100, 100)
-    y = torch.Tensor(size=input_shapes[1]).uniform_(-100, 100)
-    z = torch.Tensor(size=input_shapes[2]).uniform_(-100, 100)
+    x = gen_rand_complex(size=input_shape, low=-100, high=100).to(torch.bfloat16)
```

Comment:
Fails with gen_rand_complex `NameError: name 'gen_rand_complex' is not defined`. Probably `gen_rand_complex ` should not be used in bert tests.